### PR TITLE
Creating a file for houston-tx

### LINKED ISSUE #13: mwuenschel hometown profile
fun things to do in my hometown
## Houston, TX
- [x] Create a branch
- [x] Create a file
- [x] Commit changes
- [x] Open pull request
- [ ] Have a discussion :+1: 
- [ ] Merge pull request

### PR DESCRIPTION
**Pull request by mwuenschel**
This pull request resolves #13

### CODE REVIEW

File: places/Houston-tx.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Houston, TX
+
+### Places to go
+
```
Comment (hertogjanduevel) [inline places/Houston-tx.md:4]: Houston we have a problem

Comment (hertogjanduevel) [inline places/Houston-tx.md]: do we?

**Comment by hertogjanduevel** [comment]
Houston, we don't have a problem

**Comment by mwuenschel** [comment]
Thanks for merging for me @hertogjanduevel

**Pull request merged**